Sources/Internal/Version.swift

```diff
@@ -14,25 +14,18 @@
 
 import Foundation
 
+import GoogleUtilities_Environment
```

Comment:
Also update Package.swift, to avoid relying upon transitive dependencies

Reply:
Done.